examples/README.md

```diff
@@ -51,6 +51,31 @@ const MyGrid = glamorous.div({
 })
 ```
 
+## Ampersands & CSS Combinators
+
+You can use an ampersand (`&`) as a reference to the parent selector. This approach can be useful for CSS combinators: adjacent sibling selectors (`+`), general sibling selectors (`~`) and child selectors (`>`).
```

Comment:
Instead of "parent selector" I think we should describe it as "generated selector" or "current selector"